src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -0,0 +1,236 @@
+    errors: PropTypes.arrayOf(PropTypes.object),
+    formValues: PropTypes.object,
+    submitSucceeded: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    submitting: false,
+    errors: [],
+    formValues: {},
+    submitSucceeded: false,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.translateNamespace = 'stripes-core.createResetPassword';
+    this.passwordMatchError = {
+      code: 'password.match.error',
+      type: 'error'
+    };
+    this.newPassword = React.createRef();
+    this.state = {
+      passwordMasked: true,
+    };
+    this.validators = {
+      confirmPassword: this.confirmPasswordFieldValidation,
+    };
+  }
+
+  componentDidMount() {
+    // Focus new-password input on mount
+    this.newPassword.current.getRenderedComponent().input.current.focus();
+  }
+
+  togglePasswordMask = () => {
+    this.setState(({ passwordMasked }) => ({
+      passwordMasked: !passwordMasked,
+    }));
+  };
+
+  confirmPasswordFieldValidation = (value, { newPassword, confirmPassword } = {}) => {
+    const isConfirmPasswordInvalid = newPassword && confirmPassword && newPassword !== confirmPassword;
+    const { errors } = this.props;
+
+    if (!isConfirmPasswordInvalid) {
+      _.remove(errors, this.passwordMatchError);
+      return;
+    }
+
+    if (!_.some(errors, this.passwordMatchError)) {
+      errors.push(this.passwordMatchError);
+    }
+  };
+
+  render() {
+    const {
+      errors,
+      handleSubmit,
+      submitting,
+      onSubmit,
+      submitSucceeded,
+      formValues: { newPassword, confirmPassword },
+    } = this.props;
+    const { passwordMasked } = this.state;
+    const submissionStatus = submitting || submitSucceeded;
+    const buttonDisabled = !_.isEmpty(errors) || submissionStatus || !(newPassword && confirmPassword);
+    const buttonLabelId = `stripes-core.${submissionStatus ? 'settingPassword' : 'setPassword'}`;
+    const passwordType = passwordMasked ? 'password' : 'text';
+    const passwordToggleLabelId = `stripes-core.button.${passwordMasked ? 'show' : 'hide'}Password`;
+
+    return (
+      <div className={styles.wrapper}>
+        <div className={styles.container}>
+          <Row center="xs">
+            <Col xs={6}>
+              <OrganizationLogo />
+            </Col>
+          </Row>
+          <Row center="xs">
+            <Col xs={6}>
+              <h1 className={styles.header}>
+                <FormattedMessage id={`${this.translateNamespace}.header`} />
+              </h1>
+            </Col>
+          </Row>
+          <Row>
+            <Form
+              className={styles.form}
+              onSubmit={handleSubmit(onSubmit)}
+            >
+              <div data-test-new-password-field>
+                <Row center="xs">
+                  <Col xs={6}>
+                    <FieldLabel
+                      htmlFor="new-password"
+                      text={<FormattedMessage id={`${this.translateNamespace}.newPassword`} />}
```

Comment:
I think we should use intl.formatMessage instead of FormattedMessage in this case.